pkg/kvm/kvm.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Intel Corporation
+
+// Package kvm automate plugging of spdk devices to a qemu instance
```

Comment:
automates
spdk-> SPDK
qemu -> QEMU
we should be consistent in using these names, maybe later we can automate this within a spellchecker?

Reply:
Reworded